cypress/integration/purchase/purchase_order_material_receipt_invoice.js

```diff
@@ -176,7 +176,6 @@ describe('Create a purchase order and Material Receipts', function() {
     cy.pressStartButton();
 
     cy.getNotificationModal(generateInvoicesNotificationModalText);
-    cy.waitUntilProcessIsFinished(); // currently needed to wait for the bell notification
     cy.getDOMNotificationsNumber().should('equal', 1);
```

Comment:
shouldn't we extract this line in a separate method? I would say to place it in the same file as getDomNotificationsNumber, as it's used here and in void_a_sales_invoice_then_invoice_the_billing-candidates_again.js? It will probably be used in other tests as well.

Reply:
I'm not sure what you mean, can you elaborate, please ?

Reply:
cy.getDOMNotificationsNumber().should('equal', 1);  - it's used one time in here and 3 times in void_a_sales_invoice_then_invoice_the_billing-candidates_again.js

Reply:
You mean something like `cy.expectNumberOfDOMNotifications(1)` ?

Reply:
Yes.

Reply:
Will add the function.

Reply:
On second though: will remove completely ` cy.getDOMNotificationsNumber` and replace it with `cy.expectNumberOfDOMNotifications` since there's no other use for it.